style.css

```diff
@@ -0,0 +1,125 @@
+body {
+  font-family: sans-serif;
+}
```

Comment:
- [ ] Please consider using the correct font family. From Figma, the font family being used for this template is `Poppins`.
You can get `Poppins` font from [font.google.com](https://fonts.google.com/) and add it to the `head` section of your `HTML` or you can import it in the CSS. Check out how to add Google font to HTML [here](https://codeburst.io/intro-to-google-fonts-for-web-developers-5559b63807d3)